youtube-video

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
-cd ~/Downloads/Videos
+
+if [ ! -d ~/Downloads/Videos ]; then
+    mkdir ~/Downloads/Videos
```

Comment:
Maybe we should use `mkdir -p` here. What do you think?
https://unix.stackexchange.com/a/49264